android/src/main/java/com/thebluealliance/androidclient/fragments/RecentNotificationsFragment.java

```diff
@@ -47,6 +48,12 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
```

Comment:
Why are there two version of this fragment? Delete one and leave the proper `RecentNotificationsFragment`